docs/api-reference/Bucket.md

```diff
@@ -127,8 +127,8 @@ __Arguments__
 
 __Returns__
 
-``list``: Return value structure has not yet been permanently decided
-Currently, it's a `tuple` of `list` objects, containing the
+```list```: Return value structure has not yet been permanently decided
```

Comment:
I don't see a reason why this file was modified.

Reply:
My understanding is that it gets modified when running `python build.py` in the `gendocs` directory.

Reply:
My old (original?) commit wasn't using the correct version of `pydoc-markdown` and so messed this up. I had to create a virtualenv to get the right version. Shouldn't be modified in the latest commit(s).